src/content_mapblock.cpp

```diff
@@ -1372,12 +1358,13 @@ void mapblock_mesh_generate_special(MeshMakeData *data,
 			}
 
 			TileSpec tile = getNodeTileN(n, p, tileindex, data);
-			tile.material_flags &= ~MATERIAL_FLAG_BACKFACE_CULLING;
-			tile.material_flags |= MATERIAL_FLAG_CRACK_OVERLAY;
+			tile.layers[0].material_flags &= ~MATERIAL_FLAG_BACKFACE_CULLING;
+			tile.layers[1].material_flags &= ~MATERIAL_FLAG_BACKFACE_CULLING;
+			tile.layers[0].material_flags |= MATERIAL_FLAG_CRACK_OVERLAY;
+			tile.layers[1].material_flags |= MATERIAL_FLAG_CRACK_OVERLAY;
```

Comment:
use loop here? (obviously also applies to all the other places in the file)
because
* the compiler will unroll them for you
* someone might want to add a third tile layer sometime